### PR TITLE
Argv1bug

### PR DESCRIPTION
**Pull request by alejandrodau**
this should fix issue #357

### CODE REVIEW

File: fades/main.py [hunk 1]
```diff
@@ -154,6 +154,18 @@ def detect_inside_virtualenv(prefix, real_prefix, base_prefix):
     return prefix != base_prefix
 
 
+def _get_normalized_args(parser):
+    # support the case when executed from a shell-bang, where all the
+    # parameters come in sys.argv[1] in a single string separated
+    # by spaces (in this case, the third parameter is what is being
+    # executed)
+    env = os.environ
+    if ('_' in env and env['_'] != sys.argv[0] and len(sys.argv) >= 1 and " " in sys.argv[1]):
```
Comment (facundobatista) [inline fades/main.py]: please remove unnecessary `()` around the comparisons

File: fades/main.py [hunk 2]
```diff
@@ -154,6 +154,18 @@ def detect_inside_virtualenv(prefix, real_prefix, base_prefix):
     return prefix != base_prefix
 
 
+def _get_normalized_args(parser):
+    # support the case when executed from a shell-bang, where all the
```
Comment (facundobatista) [inline fades/main.py]: please add a docstring to the function

File: fades/main.py (hunk 1 above)
Comment (alejandrodau) [reply fades/main.py]: Done

File: fades/main.py (hunk 2 above)
Comment (alejandrodau) [reply fades/main.py]: Done

**Comment by facundobatista** [comment]
Thank you very much!!!

**Pull request merged**